setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(
+    name='AirCan',
+    packages=['aircan'],
+    version=open('VERSION').read(),
+    description='',
+    author='Datopian',
+    author_email='info@datopian.com',
```

Comment:
In the Giftless repo, this was set to the email of the original committer at Datopian. I'm not sure what's the best option here.